src/client/MainPage.react.js

```diff
@@ -20,22 +22,26 @@ function initConnection(userId: string) {
   return socket;
 }
 
+  // TODO: proper typing for conn
```

Comment:
i think this may not be possible as the socket.io library is not typed with Flow